recipes/splits/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{%set name = "splits" %}
+{%set version = "0.1.1" %}
+{%set compress_type = "tar.gz" %}
+{%set hash_type = "sha256" %}
+{%set hash_val = "b87dbd579c72e65200a12c7bdf6b01707ec368d2851fa2f5b0ee7a7983bf9fda" %}
+{%set build_num = "0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.{{ compress_type }}
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ compress_type }}
+  {{ hash_type }}: {{ hash_val }}
+
+build:
+  skip: True  # [py3k]
```

Comment:
How do we know this does not support Python 3.x? We should ask upstream to get some trove classifiers to make this clearer as to what Python versions are supported.